x-pack/platform/test/serverless/config.base.ts

```diff
@@ -184,7 +185,7 @@ export default async () => {
     // Used by FTR to recognize serverless project and change its behavior accordingly
     serverless: true,
 
-    services: {}, // define later
+    services: svlServices,
```

Comment:
config without services loaded was failing when Cypress tests started with FTr were trying to use `KibanaServer` call